src/components/component/Date.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import Date from "./Date";
+
+describe("testing", () => {
+  it("Only test for app", () => {
+    render(<Date />);
+
+    const input = screen.getByLabelText("input");
+    const button1 = screen.getByLabelText("button-1");
+    const button2 = screen.getByLabelText("button-2");
+
+    fireEvent.change(input, {
+      target: { value: ["2022-01-10", "2022-02-12", "2022-01-14"] },
+    });
+    fireEvent.change(button2, { target: { value: ["2022-02-12"] } });
+    fireEvent.change(button1, { target: { value: ["2022-01-14"] } });
```

Comment:
the test needs to be a little bit more robust, check out the acceptance criteria in the rubric to see how to build it out more.